misc/vm_configs/xmls/config-xmls/apl-mrb/sdc_launch_1uos_aaag.xml

```diff
@@ -12,7 +12,7 @@
 		<pcpu_id></pcpu_id>
 	</cpu_affinity>
 	<shm_regions desc="List of shared memrory regions for inter-VM communication.">
```

Comment:
"memory" is misspelled, but save a fix for a post-2.2 release change to how these XML documentation comments are handled

Reply:
fixed.